l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -25,8 +29,11 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
     /// @notice the asset id of Eth
     bytes32 internal immutable ETH_TOKEN_ASSET_ID;
 
+    /// @dev The chain id of L1, this contract will be deployed on multiple layers.
+    uint256 public immutable L1_CHAIN_ID;
```

Comment:
Won't this break the storage layout of current bridgehub? 
ETH_TOKEN_ASSET_ID should be moved lower as well

Reply:
Its an immutable so no